lib/java/ydoc-server/src/main/java/org/enso/ydoc/Ydoc.java

```diff
@@ -0,0 +1,68 @@
+package org.enso.ydoc;
+
+import java.io.IOException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import org.enso.ydoc.polyfill.ParserPolyfill;
+import org.enso.ydoc.polyfill.web.WebEnvironment;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.Source;
+import org.graalvm.polyglot.io.IOAccess;
+
+public final class Ydoc implements AutoCloseable {
+
+  private static final String YDOC_SERVER_PATH = "/ydocServer.js";
```

Comment:
Please rather store the `.js` file next to this class in the `org.enso.ydoc` package.

Reply:
okay